subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -181,19 +239,13 @@ protected FileSystemOperations getFileSystemOperations() {
     @Internal
     @VisibleForTesting
     ListenerBroadcast<TestListener> getTestListenerBroadcaster() {
```

Comment:
💅 I don't think exposing these `ListenerBroadcast` instances to the test is paying off anymore. Removing them would allow the state to be simplified even further.

Reply:
To have it as local state.

Reply:
Done!